File: engine/engine-model/src/main/scala/tools/packet/message/WhisperReply.scala
package tools.packet.message

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class WhisperReply(private var _target: String, private var _reply: Byte) extends PacketInput {
  def target: String = _target

  def reply: Byte = _reply

  override def opcode(): SendOpcode = SendOpcode.WHISPER
}

File: engine/engine-model/src/main/java/tools/packet/message/WhisperReply.java
package tools.packet.message;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record WhisperReply(String target, Byte reply) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.WHISPER;
   }
}